json/Explain_Element_Booster.txt

```diff
@@ -1607,7 +1607,7 @@
 		"jp_text": "エクストレセプター",
 		"tr_text": "Extreceptor",
 		"jp_explain": "アルター、フリクト系と\n特定の特殊能力継承率を\n大幅に上昇　全+10",
-		"tr_explain": "Boosts Flict/Alter-type\nability transfer rates.\nAll +10"
+		"tr_explain": "Boosts Flict, Alter and\ncertain other special\nability transfer rates.\nAll +10"
```

Comment:
Ext also boosts Stat 4 and 5, I think that's what the description's alluding to.